packages/bits/src/lib/wizard-v2/wizard-step-header/wizard-step-header.component.ts

```diff
@@ -128,4 +151,13 @@ export class WizardStepHeaderComponent extends CdkStepHeader implements AfterVie
             [STEP_STATE.ERROR]: this.wizardConfig.stepState?.error,
         }
     }
+
+    private detectLongLabel(): void {
+        if (this.labelEl) {
+            const el: HTMLElement = this.labelEl.nativeElement;
+
+            this.isLabelOverflow = el.scrollWidth > el.offsetWidth;
```

Comment:
Do we know for sure whether UX doesn't want us to show the label on hover regardless of the overflow status? It would be nice to not have to worry about making this calculation and just have the label available at all times regardless of the length.

Reply:
User could pass tooltip as input and he will be able see tooltip regardless of the length. This calculation for case when user doesn't specify tooltip and label text overflows container.

Reply:
Okay, in that case, we may want to ask UX whether we should show the tooltip at all times regardless of whether the `tooltip` input is used. The current behavior of showing a tooltip at all times on hover when the `tooltip` input is used but only showing it when the label text overflows when the tooltip input isn't being used may be seen as inconsistent from the consumer and/or end-user point of view.

This brings to mind an additional question: Maybe the consumer-provided template itself should be responsible for showing the tooltip (instead of exposing a separate `tooltip` input. The purpose of template injection is to allow consumers to put whatever they want in a certain spot on the DOM. So, in this case if they want a tooltip on their template, they can just inject a nui-tooltip along with the label contents. That way, the label template and its associated tooltip are all in one spot instead of having a special input for it. I think I like that better. What do you think?